pip/build_manylinux.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Is to be executed in the Manylinux Docker container.
+# https://github.com/pypa/manylinux
+
+# Generates the manylinux .whl file.
+# Input:
+#   "/io" is mapped to this repo dir "qsharp" (from YAML, from repo dir  "docker run -v `pwd`:/io").
+#   Environment (from YAML
+#   `docker run --env VER_MAJOR=$(Ver.Major) --env VER_MINOR=$(Ver.Minor) --env VER_PATCH=$(Ver.Patch) \
+#   --env ARCH=x86_64`):
+#       VER_MAJOR: $(Ver.Major)
+#       VER_MINOR: $(Ver.Minor)
+#       VER_PATCH: $(Ver.Patch)
+#       ARCH: Architecture: x86_64 or aarch64, etc.
+
+set -x -e
+
+# Install Rust:
+curl --proto '=https' --tlsv1.2 -sS https://sh.rustup.rs > rustup-init.sh
```

Comment:
In theory we should be using the internal Rust toolchain (https://eng.ms/docs/more/rust/intro/building) - is there a way we can use it from the container?

Reply:
I believe so, yes, we can use MS internal Rust toolchain from the container. See "Using msrustup in CI without RustInstaller" [here](https://eng.ms/docs/more/rust/intro/building/ado-pipelines). More details will follow when I proceed to that task.